From ab7537e5c220f3d1dc8c89a35b5e1e35e7c4e58c Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 4 Aug 2023 10:34:41 +0200
Subject: [PATCH 1027/1695] scripts: fix code style issues in render_font.py

Fixes code style issues reported by pycodestyle.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 6b01ba4a344b1cf595882076dcd88d24dc71c52c scripts: fix code style issues in render_font.py]
Fixes: c020eee42afd ("scripts: render_font.py use PIL")
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I539bd03a6f06797b016d75cc4e6f9a59736c547a
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/391779
Reviewed-by: Yannick FERTRE <yannick.fertre@foss.st.com>
Tested-by: Yannick FERTRE <yannick.fertre@foss.st.com>
Domain-Review: Yannick FERTRE <yannick.fertre@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 scripts/render_font.py | 39 +++++++++++++++++++++++----------------
 1 file changed, 23 insertions(+), 16 deletions(-)

diff --git a/scripts/render_font.py b/scripts/render_font.py
index 479b57c37..7593283a6 100755
--- a/scripts/render_font.py
+++ b/scripts/render_font.py
@@ -8,6 +8,7 @@ from PIL import ImageFont
 from PIL import Image
 import sys
 
+
 def get_args():
     from argparse import ArgumentParser
 
@@ -24,6 +25,7 @@ def get_args():
                         help='Print informational messages')
     return parser.parse_args()
 
+
 def c_hex(bstr):
     s = []
     for n, x in enumerate(bstr):
@@ -39,6 +41,7 @@ def c_hex(bstr):
 
     return "".join(s)
 
+
 header_template = """/*
  * This file is auto generated with
  * {cmd_line}
@@ -67,14 +70,15 @@ font_letter_template = """\t{{ letter_{id}, sizeof(letter_{id}), {width}}},
 """
 
 font_template = """const struct font font_{name} = {{
-        .first = 0x{first:02x},
-        .last = 0x{last:02x},
-	.letters = letters,
-	.height = {height},
-	.max_width = {max_width},
+    .first = 0x{first:02x},
+    .last = 0x{last:02x},
+    .letters = letters,
+    .height = {height},
+    .max_width = {max_width},
 }};
 """
 
+
 def main():
     args = get_args()
 
@@ -92,15 +96,15 @@ def main():
     text_height = font.font.height + 2
 
     for x in letters:
-        letter= chr(x)
+        letter = chr(x)
         (width, height), (offset_x, offset_y) = font.font.getsize(letter)
-        text_width = width + 2 + offset_x;
+        text_width = width + 2 + offset_x
         letter_img = Image.new("1", (text_width, text_height), 0)
         draw = ImageDraw.Draw(letter_img)
-        draw.fontmode="1"
+        draw.fontmode = "1"
         draw.text((0, 0), letter, font=font, fill=1)
-        c_letters[x] = {'img' : letter_img.tobytes(),
-                        'width' : text_width }
+        c_letters[x] = {'img': letter_img.tobytes(),
+                        'width': text_width}
 
     if args.verbose:
         print("Writing " + out_dir + "/" + font_name + ".c")
@@ -109,7 +113,8 @@ def main():
         f.write(header_template.format(cmd_line=' '.join(sys.argv)))
         f.write('#include "font.h"\n\n')
         for x in letters:
-            f.write(letter_template.format(letter=chr(x), id="{:02x}".format(x),
+            f.write(letter_template.format(letter=chr(x),
+                                           id="{:02x}".format(x),
                     c_hex=c_hex(c_letters[x]['img'])))
 
         f.write("static const struct font_letter letters[] = {\n")
@@ -119,17 +124,19 @@ def main():
         f.write("};\n\n")
 
         f.write(font_template.format(name=args.font_name,
-            first=letters[0],
-            last=letters[-1],
-            height=text_height,
-            max_width=max(l['width'] for l in c_letters.values())))
+                first=letters[0],
+                last=letters[-1],
+                height=text_height,
+                max_width=max(i['width'] for i in c_letters.values())))
 
     if args.verbose:
         print("Writing " + out_dir + "/" + font_name + ".h")
 
     with open(out_dir + "/" + font_name + ".h", 'w+') as f:
         f.write(header_template.format(cmd_line=' '.join(sys.argv)))
-        f.write(h_file_template.format(font_name=font_name, FONT_NAME=font_name.upper()))
+        f.write(h_file_template.format(font_name=font_name,
+                                       FONT_NAME=font_name.upper()))
+
 
 if __name__ == "__main__":
     main()
-- 
2.47.2

